RecipeMatch/View Controllers/ProfileViewController.m

```diff
@@ -22,47 +22,49 @@ @interface ProfileViewController () <UICollectionViewDelegate, UICollectionViewD
 @property (nonatomic, strong) UIRefreshControl *refreshControl;
 @end
 
-@implementation ProfileViewController
+static const float CORNER_RADIUS = 15;
+static const float MIN_LINE_SPACING = 10;
+static const float HEIGHT_FACTOR = 1.2;
 
+@implementation ProfileViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     
+    // setup scroll refresh
     self.refreshControl = [[UIRefreshControl alloc] init];
     [self.refreshControl addTarget:self action:@selector(fetchRecipes) forControlEvents:UIControlEventValueChanged];
     self.recipesCollectionView.refreshControl = self.refreshControl;
     
+    // setup logout button
     UIBarButtonItem *logout = [[UIBarButtonItem alloc]
                                    initWithTitle:@"Logout"
                                    style:UIBarButtonItemStylePlain
                                    target:self
                                    action:@selector(logoutBtn:)];
     self.navigationItem.leftBarButtonItem = logout;
-
     [self fetchRecipes];
 }
 
+// called after returning from PreferencesViewController
 -(void)viewWillAppear{
     [super viewDidLoad];
     [self fetchRecipes];
 }
 
-- (void) fetchRecipes{
+- (void)fetchRecipes{
     [APIManager fetchSavedRecipes:^(NSArray *recipes, NSError *error) {
         if(recipes){
             self.recipes = recipes;
             [self.recipesCollectionView reloadData];
             [self.refreshControl endRefreshing];
-        } else {
-            NSLog(@"😫😫😫 Error getting home timeline: %@", error.localizedDescription);
```

Comment:
for now, you should still support this case with at least stopping the spinner.
but for future you might want to add better error support for this case